.golangci.yml

```diff
@@ -31,6 +31,17 @@ run:
 
 issues:
   exclude-use-default: false
+
+  exclude-rules:
+    - path: ./
+      text: 'U1000: .* is unused'
```

Comment:
That one is interesting. Most of the times it _is_ helpful. There is only one generic false positive (related to gRPC server's methods and entry points), and I already reported it. All other reports are accurate, even if the they look like false positives. So I recommend to not ignore it, and just re-check it manually for a time being.